frontend/src/components/teacher/student-management/classroom-summary/ClassroomPopover.tsx

```diff
@@ -14,50 +18,76 @@ import DeleteClassroomModal from "./DeleteClassroomModal";
 
 interface ClassroomPopoverProps {
   classId: string;
+  isArchived?: boolean;
 }
 
 const ClassroomPopover = ({
   classId,
+  isArchived,
 }: ClassroomPopoverProps): React.ReactElement => {
   const {
     isOpen: isPopoverOpen,
     onOpen: onPopoverOpen,
     onClose: onPopoverClose,
   } = useDisclosure();
-  const [showDeleteModal, setShowDeleteModal] = React.useState(false);
-  const [showEditModal, setShowEditModal] = React.useState(false);
+  const [showEditModal, setShowEditModal] = useState(false);
```

Comment:
should we use `useDisclosure`? i think for the most part i updated our codebase to use that now